src/service/resources-impl.js

```diff
@@ -258,7 +258,8 @@ export class ResourcesImpl {
       try {
         this.intersectionObserver_ = new IntersectionObserver(
           (e) => this.intersect(e),
-          {root, rootMargin: '200% 25%'}
+          // rootMargin matches size of loadRect: (150vw 300vh) * 1.25.
+          {root, rootMargin: '250% 32%'}
```

Comment:
25 * 1.25 --> 31.25?
```suggestion
          {root, rootMargin: '250% 31.25%'}
```